s2i/assemble

```diff
@@ -25,7 +45,7 @@ go get -v ./...
 
 echo "---> Building application source..."
 go build -v -o main
-go install main
+cp ./main $GOBIN
```

Comment:
Why the copy instead of "go install"? And actually, I didn't realize it while writing the code initially, but "go install" should cover both the build and placement of the binary in the right location. Hm.